docs/source/user_guide/runtime-image-conf.md

```diff
@@ -189,16 +189,23 @@ The name and tag of an existing container image in a container registry that mee
 
 Example:
 
-- `jdoe/my-image:1.0.0`
+- `jdoe/my-image:1.0.0`  
+OR
+- `jdoe/my-image@sha256:ee783a4c0fccc7317c150450e84579544e171dd01a3f76cf2711262aced85767`
 
-Providing only `owner/image:tag` uses default registry: Docker Hub registry
+Providing only `owner/image[@SHA256]:tag/hash` uses default registry: Docker Hub registry
 
 In general for other public container registries, the URL shall contain also `registry`, therefore the complete URL to be used in this case is: `registry/owner/image:tag`
```

Comment:
After that sentence, since we focus on tag vs SHA digest format here, we would add a paragraph:

Use SHA256-digest format in airgapped Openshift environments to support [ImageContentSourcePolicy](https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/preparing_to_deploy_in_a_disconnected_environment/creating-operator-imagecontentsourcepolicy_rhocs) and automatic referral of images to a target repository server or to just support unique non-modifiable image digests instead of potentially mutable tags. SHA256-digest format is supported on all platforms for image references.